11.0.Dockerfile

```diff
@@ -27,6 +27,17 @@ ONBUILD ARG PGUSER=odoo
 ONBUILD ARG PGPASSWORD=odoopassword
 ONBUILD ARG PGHOST=db
 ONBUILD ARG PGDATABASE=prod
+ONBUILD ARG ODOO_MAX_CRON_THREADS=1
+ONBUILD ARG ODOO_WORKERS=4
+ONBUILD ARG ODOO_LIMIT_TIME_REAL=120
+ONBUILD ARG ODOO_LIMIT_TIME_CPU=60
+ONBUILD ARG ODOO_LIMIT_MEMORY_HARD=805306368
+ONBUILD ARG ODOO_LIMIT_MEMORY_SOFT=671088640
+ONBUILD ARG ODOO_LIMIT_REQUEST=8192
+ONBUILD ARG ODOO_LIMIT_TIME_REAL_CRON=-1
+ONBUILD ARG ODOO_DB_FILTER=""
+ONBUILD ARG ODOO_LIST_DB=1
+ONBUILD ARG ODOO_LOG_LEVEL=info
```

Comment:
You forgot to translate these `ARG` into `ENV`, like in the v8 dockerfile. That's [the cause of v11 failure](https://travis-ci.org/Tecnativa/docker-odoo-base/jobs/317022563#L4426).